app/controllers/flood_risk_engine/enrollments/addresses_controller.rb

```diff
@@ -0,0 +1,107 @@
+require_dependency "flood_risk_engine/application_controller"
+
+module FloodRiskEngine
+  module Enrollments
+    class AddressesController < ApplicationController
```

Comment:
This functionality is not accessed in the normal step way - you click on a link, rather than submitting a form.

It also needs to be accessible from different places - there is a requirement for the link to be available on the post code page if the post code can't be found.

Stepping out of the steps and then stepping back in seemed like the simplest way of dealing with this complexity.